apps/emqx_dashboard/priv/emqx_dashboard_schema.en.i18n.yml

```diff
@@ -0,0 +1,68 @@
+listeners:
+  lable: "Listeners"
+  desc:
+  "HTTP(s) listeners are identified by their protocol type and are
+  used to serve dashboard UI and restful HTTP API.<br>
+  Listeners must have a unique combination of port number and IP address.<br>
+  For example, an HTTP listener can listen on all configured IP addresses
+  on a given port for a machine by specifying the IP address 0.0.0.0.<br>
+  Alternatively, the HTTP listener can specify a unique IP address for each listener,
+  but use the same port."
```

Comment:
We'd better not put the `label` and `desc` fields here.

Doing so we make this i18n framework more general, i.e. it can be used not only for generating label and desc fields, but also for replacing any strings in the map.

Like:

```
listeners_label: "Listeners"
listeners_desc: "..."
```

And the map in the .erl:
```
{ label => i18n("listeners_label")
  desc => i18n("listeners_desc")
}
```